style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  height: 100vh;
+  background-color: white;
+  background-image: url("images/header-illsutration-mobile@2x.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.container {
+  margin: 20px;
+  margin-top: 40px;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.barItem {
+  margin: 0;
+  padding: 0;
+}
+
+.container h2 {
+  color: black;
+  font-style: normal;
+  font-family: sans-serif, 'Inter';
```

Comment:
- [ ] Good job adding the fonts of the Figma design 👍🏼, but when you deploy your project it is not going to work, please add the link to import the fonts within the `<head>` in your .html file, I suggest you use [Google Fonts](https://fonts.google.com/). 🦕 
If you inspect your project in the browser in the `computed` tab you will see the rendered font. 

![Screenshot from 2022-10-26 20-09-45](https://user-images.githubusercontent.com/91301423/198167911-f4991820-5504-42e3-aecb-5493c14e956d.png)